src/fctGraph/Utils/mutators/addPushOutFctForOutSlotWhichHasNone.js

```diff
@@ -50,7 +50,7 @@ function addPushOutFctForOutSlotWhichHasNone(fctGraph, outSlot, opts = DEFAULT_O
   const inSlotData = {
     type: InSlot.TYPE,
     dataType: outSlot.dataType,
-    name: 'Value',
+    name: outSlot.name,
```

Comment:
only concern is if the outslot is poorly named (like 'temperature out') and the name carries over leading to a misleading/confusing name. If the outslot's name is not crap though, shouldn't be a problem, but we don't always control the outslot's name :(

Reply:
Those names will not be exposed to the customer most likely anyway. 

And for the other slots (e.g. the input slot on a heater) we can't do anything about a poor name, because stream names aka slot names are becoming identifiers.